package.json

```
@@ -64,7 +67,7 @@
     "axios": "0.27.2",
     "cfenv": "1.2.4",
     "copyfiles": "2.4.1",
-    "di-ipv-cri-common-express": "alphagov/di-ipv-cri-common-express.git#v0.0.18",
+    "di-ipv-cri-common-express": "alphagov/di-ipv-cri-common-express.git#feat/add-google-tag-manager",
```

ready for the next release, I think its 19
```suggestion
    "di-ipv-cri-common-express": "alphagov/di-ipv-cri-common-express.git#v0.0.19",
```

---

Updated with published release